docs/config/markdown-math-filter.sed

```diff
@@ -0,0 +1,25 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Doxygen filter to format markdown math for Doxygen
+#
+# In Doxygen documentation, including markdown files or Jupyter notebooks that
+# get converted to markdown, you can use standard $...$ syntax for inline math,
+# and the following syntax for display-style math:
+#
+# $$\begin{equation}
+# ...
+# \end{equation}$$
+#
+# You can also replace `equation` with another supported environment, e.g.
+# `align`.
+#
+# This Doxygen filter formats the math for Doxygen, i.e., uses \f$...\f$ for
+# inline math and \f{equation}{ ... \f} for display-style math.
+
+# Wrap display math in \f{equation}{ ... \f}
+s|\$\$\\begin{\([^}]*\)}|\\f{\1}{|g
+s|\\end{\([^}]*\)}\$\$|\\f}|g
+
+# Wrap inline math in \f$...\f$
+s|\([^\\f]\)\$\([^$]*\)\$|\1\\f\$\2\\f\$|g
```

Comment:
This is too aggressive.  From the "Build System" guide:
```
add_spectre_library(\f${LIBRARY} \f${LIBRARY_SOURCES})
```
Similar things happen in a few other places.

I don't have any ideas for how to fix this without support in the markdown renderer.

Reply:
Good catch. How about we don't apply the math rendering in code blocks (fenced with backticks)? That's probably how markdown renderers handle this as well.

Reply:
Yes, that should do it.